### PR TITLE
params: set mainnet terminal total difficulty for the merge

### PR DESCRIPTION
**Pull request by MariusVanDerWijden**
This PR sets the mainnet total terminal difficulty as defined inhttps://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/paris.md
This enables the merge on mainnet. 

```
INFO [08-15|17:25:47.999]  
INFO [08-15|17:25:47.999] --------------------------------------------------------------------------------------------------------------------------------------------------------- 
INFO [08-15|17:25:47.999] Chain ID:  1 (mainnet) 
INFO [08-15|17:25:47.999] Consensus: Beacon (proof-of-stake), merged from Ethash (proof-of-work) 
INFO [08-15|17:25:47.999]  
INFO [08-15|17:25:48.000] Pre-Merge hard forks: 
INFO [08-15|17:25:48.000]  - Homestead:                   1150000  (https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/homestead.md) 
INFO [08-15|17:25:48.000]  - DAO Fork:                    1920000  (https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/dao-fork.md) 
INFO [08-15|17:25:48.000]  - Tangerine Whistle (EIP 150): 2463000  (https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/tangerine-whistle.md) 
INFO [08-15|17:25:48.000]  - Spurious Dragon/1 (EIP 155): 2675000  (https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/spurious-dragon.md) 
INFO [08-15|17:25:48.000]  - Spurious Dragon/2 (EIP 158): 2675000  (https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/spurious-dragon.md) 
INFO [08-15|17:25:48.000]  - Byzantium:                   4370000  (https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/byzantium.md) 
INFO [08-15|17:25:48.000]  - Constantinople:              7280000  (https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/constantinople.md) 
INFO [08-15|17:25:48.000]  - Petersburg:                  7280000  (https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/petersburg.md) 
INFO [08-15|17:25:48.000]  - Istanbul:                    9069000  (https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/istanbul.md) 
INFO [08-15|17:25:48.000]  - Muir Glacier:                9200000  (https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/muir-glacier.md) 
INFO [08-15|17:25:48.000]  - Berlin:                      12244000 (https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/berlin.md) 
INFO [08-15|17:25:48.000]  - London:                      12965000 (https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/london.md) 
INFO [08-15|17:25:48.000]  - Arrow Glacier:               13773000 (https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/arrow-glacier.md) 
INFO [08-15|17:25:48.000]  - Gray Glacier:                15050000 (https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/gray-glacier.md) 
INFO [08-15|17:25:48.000]  
INFO [08-15|17:25:48.000] Merge configured: 
INFO [08-15|17:25:48.000]  - Hard-fork specification:   https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/paris.md) 
INFO [08-15|17:25:48.000]  - Total terminal difficulty: 58750000000000000000000 
INFO [08-15|17:25:48.000]  - Merge netsplit block:      <nil> 
INFO [08-15|17:25:48.000] --------------------------------------------------------------------------------------------------------------------------------------------------------- 
INFO [08-15|17:25:48.000]  
```

### CODE REVIEW

File: params/config.go [hunk 1]
```diff
@@ -471,10 +474,10 @@ func (c *ChainConfig) String() string {
 	// Add a special section for the merge as it's non-obvious
 	if c.TerminalTotalDifficulty == nil {
 		banner += "The Merge is not yet available for this network!\n"
-		banner += " - Hard-fork specification: https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/paris.md)"
+		banner += " - Hard-fork specification: (https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/paris.md)"
 	} else {
 		banner += "Merge configured:\n"
-		banner += " - Hard-fork specification:    https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/paris.md)\n"
+		banner += " - Hard-fork specification:    (https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/paris.md)\n"
```
Comment (karalabe) [inline params/config.go:480]: Please revert, the `Hard-fork specification` *is* the link

**Review by holiman — APPROVED** [review]
LGTM!

File: params/config.go (hunk 1 above)
Comment (karalabe) [reply params/config.go:480]: Ah, please delete the closing paranthesis. That's the extra.

**Comment by junderw** [comment]
Good job, y'all. I wish you the best of luck on the Merge.

**Comment by alephile** [comment]
LFG!

**Comment by joanyin-bolt** [comment]
Bravo!

**Comment by heliocosta1337** [comment]
🐼🔥

**Comment by Tadouh** [comment]
Round of applause for all the eth devs and consensus client teams working many years to get this done, and get it done right! Thanks for working your butts off and not rushing this like most the people who have no skin in the game wanted you to.

**Pull request merged**